modules/imgur.py

```diff
@@ -1,14 +1,18 @@
 #--require-config imgur-api-key
 
-import re
-from src import ModuleManager, utils
+import re, json
+from src import ModuleManager, utils, EventManager, Logging
+from datetime import datetime
```

Comment:
do `import datetime` and refer to it as `datetime.datetime.utcfromtimestamp` - just a code style consistency thing